gw_eccentricity/eccDefinition.py

```diff
@@ -272,6 +279,29 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 eccentricity is the cause, and set the returned eccentricity
                 and mean anomaly to zero.
                 USE THIS WITH CAUTION!
+
+            omega_gw_extrema_interpolation_method : str, default="rational_fit"
+                Specifies the method used to build the interpolations for 
+                `omega_gw_pericenters_interp(t)` or `omega_gw_apocenters_interp(t)`.
+                The available options are:
+
+                - `spline`: Uses `scipy.interpolate.InterpolatedUnivariateSpline`.
+                    - Best suited for cleaner data, such as when waveform modes are generated
+                        using models like SEOB or TEOB.
+                    - Faster to construct and evaluate.
+                    - Since it fits through every data point, it may exhibit oscillatory 
+                        behavior, particularly near the merger.
```

Comment:
near the merger, especially for noisy NR data.

Reply:
Done